packages/eslint-config/rules/base.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+  rules: {
+    // We use global requires in various places, e.g. code splitting instances.
+    'global-require': 0,
+    // Ignore linebreak style. The CRLF / LF endings wont matter if a windows user correctly
+    // converts CRLF to LF upon commits; otherwise there are errors every line.
+    'linebreak-style': 0,
+    // Other Ueno rules
+    'padded-blocks': 0,
+    'consistent-return': 0,
+    'no-confusing-arrow': 0,
+    'import/no-named-as-default': 0,
+    'import/no-extraneous-dependencies': 0,
+    'class-methods-use-this': 0,
+    // Allow useful console logging
+    'no-console': ['error', {
+      allow: ['info', 'warn', 'error']
+    }],
+    // Allow ++ in for loops
+    'no-plusplus': ['error', {
+      allowForLoopAfterthoughts: true
+    }],
+    // Allow reassigning props of objects, e.g. `foo.bar = 2`
+    'no-param-reassign': ['error', {
+      props: false
+    }],
+    'function-paren-newline': ['error', 'consistent'],
+    'object-curly-newline': ['error', { consistent: true }],
+  },
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    jest: true
+  }
+}
```

Comment:
Missing newline, we need to add linting to this project 🙃

Reply:
I think we can make the project lint itself using a relative path. The main reason xo is in there it's because this codebase was derived from [my module boilerplate](https://github.com/gakimball/module). But hey, might as well have our linting rules lint themselves, right?

Reply:
Oh, and either way I just realized one of the projects isn't running the linter in its `npm test` command. Check the Travis log here: <https://travis-ci.org/ueno-llc/styleguide/jobs/290491473> One package runs just `mocha`, one runs `xo && mocha`. They should both read `xo && mocha` (or `eslint && mocha` if we switch).

Reply:
yeah, I removed xo earlier because of the errors, was gonna add eslint with a relative path